relayer/src/chain/substrate.rs

```diff
@@ -624,8 +626,36 @@ impl ChainEndpoint for SubstrateChain {
         Ok((event_receiver, monitor_tx))
     }
 
+    fn init_beefy_monitor(
```

Comment:
maybe init beefy monitor can move init_event, so we not need have this function in ChainEndpoint trait.